auth/client/src/test/java/org/wildfly/security/auth/client/ElytronXmlParserTest.java

```diff
@@ -2,22 +2,31 @@
 
```

Comment:
Not really related to this PR but maybe could be fixed here, it looks like this test is missing a copyright header.

Reply:
Fixed